.github/workflows/ci-repo2docker.yaml

```diff
@@ -1,25 +1,14 @@
 name: repo2docker CI
 
-on: [push, pull_request]
+name: Binder
+on: [push]
```

Comment:
Do we want to drop `pull_request` here?

Reply:
I think so yeah. Not sure how it adds any value since there is already a test that tries to run all the notebooks.

Reply:
The build and deploy action builds the Sphinx site, and the repo2docker action builds the binder image. I think those are both valuable things to do for all PRs.

I didn't think the repo2docker action actually ran the notebooks either?